specs2-akka/src/main/scala/com/dwolla/testutils/akka/AkkaTestKitSpecs2Support.scala

```diff
@@ -13,5 +13,9 @@ class AkkaTestKitSpecs2Support(config: Option[Config] = None)(implicit execution
   def after: Unit = TestKit.shutdownActorSystem(system)
 
   private def safeTypeName: String = enclosingClassName(this.getClass)
-  @tailrec private def enclosingClassName(clazz: Class[_]): String = if (clazz.isLocalClass) enclosingClassName(clazz.getEnclosingClass) else clazz.getSimpleName
+  @tailrec private def enclosingClassName(clazz: Class[_]): String =
+    if (clazz.isLocalClass || clazz.isAnonymousClass)
```

Comment:
On Java 8, `clazz.isLocalClass` returns true, but on Java 11, it returns false. The opposite seems to be true for `clazz.isAnonymousClass`. Using `||` to combine them should work on either platform.

Reply:
Is there a specific reason why this is the case? It seems like such a weird thing to change in between versions.